.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -1,6 +1,6 @@
 name: Bug Report
 description: File a bug report
-title: ''
+title: Title
```

Comment:
According to https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms#top-level-syntax you can omit the key entirely?

Reply:
So you can't leave it blank but you can remove it entirely? Thanks, GitHub, for the totally expected behavior!